packages/frontend/src/components/ProjectConnection/formContext.ts

```diff
@@ -0,0 +1,7 @@
+import { createFormContext, type UseFormReturnType } from '@mantine/form';
+import type { ProjectConnectionForm } from './types';
+
+export type Form = UseFormReturnType<ProjectConnectionForm>;
+export type FormInputProps = ReturnType<Form['getInputProps']>;
+export const [FormProvider, useFormContext, useForm] =
```

Comment:
I was wondering if we should be using explicit names for these.. like

`ProjectFormProvider`
`useProjectFormContext`
`useProjectForm`

Reply:
but im happy as is 🤌